src/yews/models/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .cpic import *
+from .polarity import *
+from .focal_mechanism import *
 
```

Comment:
This is OK now. But we will later want to selectively import model constructors into __init__.py so that we can  the limit usage of model classes into restricted ways that has been tested.